### PR TITLE
Cert error copy update

### LINKED ISSUE #1043: Wording of CertErrorComponent.tsx error message
Current [error message](https://github.com/konveyor/mig-ui/blob/c35f58e1f52978ffe85c24e27c823671bbf89e63/src/app/auth/CertErrorComponent.tsx#L17) of CertErrorComponent.tsx:

```
       <div>
            A certificate error has occurred, likely due to the usage of self signed certificates in
            one of the clusters. Please try to visit the failed url and accept the CA:
            <a href={this.props.certError.failedUrl}> {this.props.certError.failedUrl}</a>
            <div />
            The correct way to address this is to install your self CA into your browser.
            <div />
            NOTE: The contents of the resulting page may report "unauthorized". This is expected.
            After accepting the certificate, please reload the app.
          </div>
```

I suggest replacing the current error message with the following version:

```
       <div>
            A certificate error has occurred, likely caused by using self-signed CA certificates in
            one of the clusters. Navigate to the following URL and accept the certificate:
            <a href={this.props.certError.failedUrl}> {this.props.certError.failedUrl}</a>
            <div />
            If an "Unauthorized" message appears after you have accepted the 
            certificate, refresh the web page.
            <div />
            To fix this issue permanently, add the certificate to your web browser's trust store.
         </div>
```

### PR DESCRIPTION
**Pull request by eriknelson**
Fixes #1043

### CODE REVIEW

**Review by ibolton336 — APPROVED** [review]

**Comment by eriknelson** [comment]
Travis continues to stick, I ran yarn ci and it's passing on my workstation. Merging.

File: src/app/auth/CertErrorComponent.tsx [hunk 1]
```diff
@@ -14,14 +14,14 @@ class CertErrorComponent extends React.Component<any> {
...
           <div>
-            A certificate error has occurred, likely due to the usage of self signed certificates in
-            one of the clusters. Please try to visit the failed url and accept the CA:
+            A certificate error has occurred, likely caused by using self-signed CA certificates in
+            one of the clusters. Navigate to the following URL and accept the certificate:
             <a href={this.props.certError.failedUrl}> {this.props.certError.failedUrl}</a>
             <div />
-            The correct way to address this is to install your self CA into your browser.
+            If an "Unauthorized" message appears after you have accepted the certificate, refresh
+            the web page.
```
Comment (apinnick) [inline src/app/auth/CertErrorComponent.tsx:22]: @eriknelson 
I just want to confirm something. Does "refresh the web page" mean that the user should reload the OAuth server URL or the MTC console?

**Comment by eriknelson** [comment]
The MTC console

**Comment by apinnick** [comment]
In that case, I would like to change that line:

 If an "Unauthorized" message appears after you have accepted the certificate, navigate to the MTC console and refresh the page.

**Comment by eriknelson** [comment]
Will post an update for your review this morning.

**Pull request merged**